package.json

```diff
@@ -14,8 +14,7 @@
   "scripts": {
     "build": "ethereumjs-config-build",
     "prepublishOnly": "npm run test && npm run build",
-    "coverage": "ethereumjs-config-coverage",
-    "coveralls": "ethereumjs-config-coveralls",
+    "coverage": "npx nyc npm run test",
```

Comment:
I'll give this a go since everything config and setup related is so much in a transition anyhow with the move to the monorepo. We nevertheless should avoid to case-by-case drop out the `ethereumjs-config` configuration unification, but rather make updates on the config library side.

This is more work in the beginning but clearly showed some benefits on having consistent configuration and at the end less work on updating script directives and the like.

We can nevertheless also have some continued respectively revived discussion on how we want to evolve here (with the config library).

Reply:
Sounds good, I agree I think there is a lot of benefit to `ethereumjs-config` but the coveralls and coverage commands in this case were not being so helpful since a) we are now using gh actions to push to coveralls so we don't need a coveralls dep and b) we will be using codecov in the monorepo for better monorepo support. c) just running `npx nyc npm run test` seems to do the trick. I think after the monorepo transition we'll have some more clarity, but nonetheless unified lint, prettier, nyc, etc. configs are super useful.